versions/gui_gui.py

```diff
@@ -0,0 +1,101 @@
+import pyperclip
+import os
+import sys
+from tkinter import *
+from tkinter import ttk
+from functools import partial
+from PIL import Image, ImageTk
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from library.TextConverter import TextConverter
+converter = TextConverter()
+text_label_column=0
+copy_button_column=1
+text_output_column=2
+dir = os.path.dirname(__file__)
+
+def copy_to_clipboard(text):
+    pyperclip.copy(text['text'])
+
+def add_text(*args):
+    new_text = translate_text.get()
+    x_size=int(8.5*len(new_text)+240)
+    root.geometry(f"{x_size}x{350}")
+    reversed_text['text'] = converter.reverse_text(new_text)
+    flipped_text['text'] = converter.text_flip(new_text)
+    enchanted_text['text'] = converter.enchant_text(new_text)
+    capitalized_text['text'] = converter.case_switch(new_text, "upper")
+    lowercased_text['text'] = converter.case_switch(new_text, "lower")
+    leetspeak_text['text'] = converter.leetspeak(new_text)
+    scrambled_text['text'] = converter.scramble_text(new_text)
+    piglatin_text['text'] = converter.piglatin(new_text)
+    zalgo_text['text'] = converter.zalgo_text(new_text)
+    emoticons_text['text'] = converter.text_to_emoticons(new_text)
+
+def add_copy_buttons():
+    ttk.Button(root, command=partial(copy_to_clipboard, reversed_text), image = copy_icon).grid(sticky = W, row=1, column=copy_button_column)
```

Comment:
You could simplify this using loops, arguably simpler than later in the code to do so.